repos.go

```diff
@@ -440,6 +440,32 @@ type Commit struct {
 	} `json:"files"`
 }
 
+// Compare represents detail of comparison between two branches or commits
+type Compare struct {
+	URL				string		`json:"url"`
+	HTMLURL			string		`json:"html_url"`
+	PERMALINKURL	string		`json:"permalink_url"`
+	DIFFURL			string		`json:"diff_url"`
+	PATCHURL		string		`json:"patch_url"`
+	BASECOMMIT		Commit		`json:"base_commit"`
+	MERGEBASECOMMIT Commit		`json:"merge_base_commit"`
+	STATUS			string		`json:"status"`
+	AHEADBY			int			`json:"ahead_by"`
+	BEHINDBY		int			`json:"behind_by"`
+	TOTALCOMMITS	int			`json:"total_commits"`
+	COMMITS			[]Commit	`json: "commits"`
+	Files 			[]struct {
+		Filename  string `json:"filename"`
+		Additions int    `json:"additions"`
+		Deletions int    `json:"deletions"`
+		Changes   int    `json:"changes"`
+		Status    string `json:"status"`
+		RawURL    string `json:"raw_url"`
+		BlobURL   string `json:"blob_url"`
+		Patch     string `json:"patch"`
+	} `json:"files"`
+}
```

Comment:
@adamcroissant Try https://mholt.github.io/json-to-go/. This file needs to be gofmt'd, GoLand/IntelliJ have an option to do this. I have Ctrl+S remapped to gofmt since IntelliJ autosaves.

Reply:
Handy tool. Thanks. I was winging it based off of the existing examples in the file.